python/bin/start.sh

```diff
@@ -62,6 +62,30 @@ fi
 if [ -n "${PY_FILES}" ]; then
   OPT_FILES="${OPT_PY_FILES},${PY_FILES}"
 fi
+#if Hbase catalog is passed, then required hbase dependency are copied to staging location and added to jars
+if [ -n "${CATALOG}" ]; then
+  echo "Downloading Hbase jar dependency"
+  wget https://repo1.maven.org/maven2/org/apache/hbase/hbase-client/2.4.12/hbase-client-2.4.12.jar
+  wget https://repo1.maven.org/maven2/org/apache/hbase/hbase-shaded-mapreduce/2.4.12/hbase-shaded-mapreduce-2.4.12.jar
+  wget https://repo1.maven.org/maven2/org/apache/htrace/htrace-core4/4.2.0-incubating/htrace-core4-4.2.0-incubating.jar
+  gsutil copy hbase-client-2.4.12.jar ${GCS_STAGING_LOCATION}/hbase-client-2.4.12.jar
+  gsutil copy hbase-shaded-mapreduce-2.4.12.jar ${GCS_STAGING_LOCATION}/hbase-shaded-mapreduce-2.4.12.jar
+  gsutil copy htrace-core4-4.2.0-incubating.jar ${GCS_STAGING_LOCATION}/htrace-core4-4.2.0-incubating.jar
+  echo "Passing downloaded dependency jars"
+  OPT_JARS="${OPT_JARS},${GCS_STAGING_LOCATION}/hbase-client-2.4.12.jar,${GCS_STAGING_LOCATION}/hbase-shaded-mapreduce-2.4.12.jar,${GCS_STAGING_LOCATION}/htrace-core4-4.2.0-incubating.jar,file:///usr/lib/spark/external/hbase-spark.jar"
+  rm hbase-client-2.4.12.jar
+  rm hbase-shaded-mapreduce-2.4.12.jar
+  rm htrace-core4-4.2.0-incubating.jar
+fi
+
+if [ -n "${HBASE_SITE_PATH}" ]; then
+  #Copy the hbase-site.xml to docker context
+  cp $HBASE_SITE_PATH .
+  export HBASE_SITE_NAME=`basename $HBASE_SITE_PATH`
+  docker build -t "${IMAGE}" -f dataproc_templates/hbase/Dockerfile --build-arg HBASE_SITE_NAME=${HBASE_SITE_NAME} .
```

Comment:
I think there is issue with docker build from Mac M1 Pro, can you research around this and add --platform which works consitently?

I added docker build --platform=linux/amd64 to make it work otherwise I get exec /entrypoint.sh: exec format error

Reply:
Thanks for bringing this. Yeah correct this is specific to Mac. Hence, it might be a separate enhancement. For now this PR is more focused on automating the process of image build. However, I will capture about this "--platform"  in Read me.

Reply:
Also I did look at it, looks like this is automatically set by the docker itself. This issue might be due to wrong installation or installation problem specific to Mac. Added link in the new commit. Ref https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope